### PR TITLE
ID-42

### PR DESCRIPTION
**Pull request by SharmaDinesh**

### CODE REVIEW

File: app/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL="https://testapi.getpaidinbitcoin.com.au"
```
Comment (ahinchliff) [inline app/.env:1]: This should be something `REACT_APP_GPIB_URL`

File: app/apis/idem.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
```
Comment (ahinchliff) [inline app/apis/idem.ts:1]: rename this file to gpib

File: app/.env (hunk 1 above)
Comment (bitcoinbrisbane) [reply app/.env:1]: We can actually get it for https://github.com/dltxio/idem/blob/master/assets/sites.json#L7

Comment (SharmaDinesh) [reply app/.env]: Done

File: app/apis/idem.ts (hunk 2 above)
Comment (SharmaDinesh) [reply app/apis/idem.ts]: Done

**Pull request closed without merging**